src/v2/Components/ArtworkFilter/index.tsx

```diff
@@ -269,21 +225,16 @@ export const BaseArtworkFilter: React.FC<
     },
   })
 
-  // If there was an error fetching the filter,
-  // we still want to render the rest of the page.
-  if (!hasFilter) return null
-
   function fetchResults() {
     toggleFetching(true)
 
-    const relayRefetchVariables = {
-      first: 30,
-      ...allowedFilters(filterContext.filters),
-      keyword: filterContext.filters!.term,
-    }
-
     const refetchVariables = {
-      input: relayRefetchVariables,
+      input: {
+        first: 30,
```

Comment:
I didn't get the change of 10 from the route to 30 here, was it because of the issue you mentioned here?